docs/en/observability/synthetics-quickstart.asciidoc

```diff
@@ -21,144 +23,111 @@ and a packaged todo application with a custom suite of tests.
 
 [discrete]
 [[synthetics-quickstart-step-one]]
-== Step 1: Clone the `elastic/synthetics` repository
-
-This template pulls the Elastic synthetics image, builds your synthetic tests,
-schedules and runs them with Heartbeat, and sends the resulting data to the Elastic Stack.
-
-Clone the https://github.com/elastic/synthetics[elastic/synthetics] repository
-and change directories into `examples/docker`:
-
-[source,sh]
-----
-git clone https://github.com/elastic/synthetics.git &&\
-cd synthetics/examples/docker
-----
+== Step 1: Pull the latest synthetics docker image
 
-There are two files in `synthetics/examples/docker` that you'll need to edit:
+Elastic Synthetics is regularly updated during this experimental phase, even within the same version.
+You'll want to ensure you're running the latest docker image by running the below command.{synthetics-image}
```

Comment:
Was the use of the `{synthetics-image}` attribute accidental?

<img width="759" alt="Screen Shot 2021-03-22 at 1 24 17 PM" src="https://user-images.githubusercontent.com/5618806/112053727-e9e90700-8b11-11eb-83bf-a5d3b1529230.png">

Reply:
No, I thought it'd be helpful to have a shortcut there.

Reply:
Actually, now that I think of it, is there a variable we can just use for heartbeat?

Reply:
I'm still not quite sure what this accomplishes?

<img width="616" alt="Screen Shot 2021-03-22 at 1 43 46 PM" src="https://user-images.githubusercontent.com/5618806/112055893-a0e68200-8b14-11eb-8251-9ff830b67599.png">

Reply:
The usage in the code block makes sense

Reply:
> Actually, now that I think of it, is there a variable we can just use for heartbeat?

There's `{heartbeat}` for Heartbeat.

Reply:
Switched to using that except for code where I think the literal value (which must be lowercase too) makes more sense